packages/deployed-backend-client/src/deployed_backend_client_factory.ts

```diff
@@ -26,14 +26,15 @@ export enum ApiAuthType {
   AMAZON_COGNITO_USER_POOLS = 'AMAZON_COGNITO_USER_POOLS',
 }
 
-export type SandboxMetadata = {
+export type ListBackendsMetadata = {
   name: string;
   lastUpdated: Date | undefined;
   status: BackendDeploymentStatus;
   backendId: BackendIdentifier | undefined;
 };
 
-export type ListSandboxesRequest = {
+export type ListBackendsRequest = {
+  deploymentType: string;
```

Comment:
I wonder if this should be an optional field and if not specified we return data from all Amplify stacks?